### PR TITLE
Fixing Store integration tests.

### PR DESCRIPTION
**Pull request by cprov**
Coping with recently added snap-name size limit (40-chars) in Store.

Pending more specific error response from the store. See LP: #1681545

### CODE REVIEW

File: integration_tests/__init__.py [hunk 1]
```diff
@@ -273,14 +273,30 @@ def list_registered(self, expected_snaps):
         process.close()
         return process.exitstatus
 
+    def get_unique_name(self, prefix='u1test'):
+        """Return an unique snap name.
```
Comment (sergiusens) [inline integration_tests/__init__.py]: a unique

Comment (cprov) [inline integration_tests/__init__.py]: Fixed in the next revision.

**Comment by cprov** [comment]
@sergiusens great opportunity to drop 'u1test' prefix, thanks to pointing it. Using 'snapcrafttest' in the next rev.

File: snapcraft/tests/fake_servers.py [hunk 2]
```diff
@@ -603,7 +607,7 @@ def _handle_register_409(self, error_code):
         self.wfile.write(json.dumps(response).encode())
 
     def _handle_register_429(self, error_code):
-        self.send_response(409)
+        self.send_response(429)
```
Comment (sergiusens) [inline snapcraft/tests/fake_servers.py]: hah

**Pull request merged**